libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/SkillMocks/SkillMock.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Builder.Dialogs.Debugging;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.SkillMocks
+{
+    public abstract class SkillMock
```

Comment:
/// <Comments Needed>

Reply:
What are examples of other skill mocks? 

---
In reply to: [448517186](https://github.com/microsoft/botbuilder-dotnet/pull/4212#discussion_r448517186) [](ancestors = 448517186)

Reply:
removed